### ISSUE TITLE
Android | isSupported returns boolean instead of 'TouchID' as it returns on IOS

### ISSUE THREAD

**Issue by ronilitman** [naoufal/react-native-touch-id#155]
Is this a bug or the right implementation you wanted? If yes, I think the readme should be updated.

**Comment by zibs** [comment]
PR's are welcome!

**Comment by gavaliswapnil** [comment]
can this lib support to face authentication for android

**Comment by saadmutahar** [comment]
@gavaliswapnil well for now this library is not supported for android.

**Comment by zibs** [comment]
@saadmutahar to clarify - it does support android fingerprint authentication, it just does not support android face id yet.

**Comment by saadmutahar** [comment]
@zibs yeah sorry I was just about to write android faceId instead of android. Thank you.

**Comment by akash4320** [comment]
Can anyone help me in letting know how to make Face ID authentication using react native then
